crates/relayer-cli/src/chain_registry.rs

```diff
@@ -168,6 +168,7 @@ where
         compat_mode: None,
         clear_interval: None,
         excluded_sequences: BTreeMap::new(),
+        allow_ccq: true,
```

Comment:
Could you also add with some comments in `config.toml`?

Reply:
Sure, if it looks good I'll finish writing the documentation and changelog entry

Reply:
Done in 17081fec528a95b0ffc8e130c4af36ce70fea8c0